docs/extend/extend-apiml/api-mediation-security.md

```diff
@@ -594,3 +645,33 @@ in following shell scripts:
 - `$ZOWE_RUNTIME/api-mediation/scripts/api-mediation-start-catalog.sh`
 - `$ZOWE_RUNTIME/api-mediation/scripts/api-mediation-start-discovery.sh`
 - `$ZOWE_RUNTIME/api-mediation/scripts/api-mediation-start-gateway.sh`
+
+## Security Service Client library
+
+The `security-service-client-spring` library enables authentication and protection using security providers.
+The library contains providers, filters and handlers as Spring components. The `security-service-client-spring` library enables any Spring client to authenticate the mainframe user credentials.
```

Comment:
"to authenticate the mainframe user credentials." please remove "the" here, unless you meant some specific user credentials.